sentry/src/main/java/io/sentry/event/interfaces/SentryStackTraceElement.java

```diff
@@ -13,6 +18,7 @@
     private final Integer colno;
     private final String absPath;
     private final String platform;
+    private final Map<String, Object> vars;
```

Comment:
I'm actually for more clarity in the code, it can swap names in the JSON, plenty of other stuff does.